pandas/core/strings.py

```diff
@@ -328,19 +328,43 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
 def str_startswith(arr, pat, na=np.nan):
     """
-    Return boolean Series/``array`` indicating whether each string in the
-    Series/Index starts with passed pattern. Equivalent to
-    :meth:`str.startswith`.
+    Test if the start of each string element matches a pattern.
+
+    Equivalent to :meth:`str.startswith`.
 
     Parameters
     ----------
-    pat : string
-        Character sequence
-    na : bool, default NaN
+    pat : str
+        Character sequence. Regular expressions are not accepted.
+    na : object, default NaN
+        Object shown if element tested is not a string.
 
     Returns
     -------
-    startswith : Series/array of boolean values
+    startswith : Series or array-like of bool
+        A Series of booleans indicating whether the given pattern matches
+        the start of each string element.
+
+    See Also
+    --------
+    str_endswith : Same as startswith, but tests the end of string.
+    str.startswith : Python standard library string method.
```

Comment:
I think there is some bug preventing the links to be generated when you build the html with the `--single` option. If you build the whole documenation `doc/make.py html`, it will take something like 5 minutes to complete, but I think you should have the links working.

Replacing `str_endswith` with `Series.str.endswith` is the right way.

Also, I'd personally add `Series.str.contains`, which also looks for the pattern, but in any position. So, I think it could be useful for some users visiting the page to know about it too.

Reply:
yep thanks.
The entire build one shows the links properly.
Added contains